ova/generate_ova.sh

```diff
@@ -63,7 +63,8 @@ build_ova() {
     OVF_VM="wazuh${OVA_VERSION}.ovf"
     OVA_FIXED="wazuh${OVA_VERSION}-fixed.ova"
     OVA_VMDK="wazuh${OVA_VERSION}-disk001.vmdk"
-    ELK_MAJOR=`echo ${ELK_VERSION}|cut -d"." -f1`
+    ELK_MAJOR=`echo ${FILEBEAT_VERSION}|cut -d"." -f1`
```

Comment:
Try to maintain  the same names, either use `ELK_MAJOR/ELK_VERSION` or `FILEBEAT_MAJOR/FILEBEAT_VERSION`

Reply:
Done in 71fd88cf61ad859245d50e315178f0058cb61754